README.md

```diff
@@ -21,6 +21,9 @@
 - Use database migration fails to maintain database schema.
 - Set up associations between models.
```

Comment:
### Optional
- [ ] Going forward, it would be better if we check typos in our commit. This is very important as committing is part of code documentation

Reply:
Thank you I'll take more care for the next!!